loader/linux/icd_linux.c

```diff
@@ -141,7 +200,19 @@ void khrIcdOsVendorsEnumerate(void)
     {
         khrIcd_free_getenv(envPath);
     }
+}
+
+// go through the list of vendors in the two configuration files
+void khrIcdOsVendorsEnumerate(void)
+{
+    khrIcdVendorsEnumerateEnv();
+
+    khrIcdOsDirEnumerate(ICD_VENDOR_PATH, "OCL_ICD_VENDORS", ".icd", khrIcdVendorAdd, 0);
+
 #if defined(CL_ENABLE_LAYERS)
+    // system layers should be closer to the driver
```

Comment:
Is it important to to distinguish these as system layers? I'm thinking that a user can choose to use the file-based mechanism for their own layers if they want to, e.g. if using environment variables is inconvenient on Android.

Reply:
Indeed the user can use them, as long as they can install the libraries in the default folder. On a classic linux distribution that would require root access, unless the user overrides the default location through environment variables. This naming was sparked from a discussion about layers that could be configured in safe mode as well on windows platform, so with a system wide scope and independent from users. We can discuss giving them a new name.

Reply:
Can we check what Vulkan is doing to see if we are consistent?